python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -175,7 +177,7 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
                 if psfCellSet:
                     psfCellSet.insertCandidate(psfCandidate)
             except Exception, e:
-                self.log.log(-2, "Skipping PSF candidate %d of %d: %s" % (i, len(psfCandidateList), e))
+                debugLog.debug(2, "Skipping PSF candidate %d of %d: %s" % (i, len(psfCandidateList), e))
```

Comment:
What does the first argument `2` mean? (Also, why the % formatting?) I would expect to see:

    self.log.debug("Skipping PSF candidate %d of %d: %s", i, len(psfCandidateList), e)